src/hotspot/os/linux/osContainer_linux.cpp

```diff
@@ -62,6 +62,10 @@ void OSContainer::init() {
   _is_containerized = true;
 }
 
+julong OSContainer::oscontainer_cache_timeout() {
+  return NANOSECS_PER_SEC / OsCachedMetricsRefreshRate;
+}
+
```

Comment:
I don't think the wrapper is needed, I'd just use the switch directly.

Reply:
I prefer to keep this wrapper instead of using in the code "NANOSECS_PER_SEC / OsCachedMetricsRefreshRate" in order to preserve an easy reading of code and to be closer to the origin name of the replaced macro - OSCONTAINER_CACHE_TIMEOUT.

I decided to use a refresh rate instead of a timeout as a command line option for easier perception/setting of values i.e.: 50 times/s instead of 20000000 ns (= 20 ms)

But if to use a timeout in ns as a command line option then such a wrapper is not needed.